dashboard/app/views/home/_privacy.md

```diff
@@ -236,34 +246,66 @@ In the event that Code.org is acquired by, combined or merged with another entit
+
+If you choose to use our Website from the European Union, please note the following information: Personal Information we collect from you will be processed in the United States. The United States has not sought nor received a finding of “adequacy” from the European Union under Article 45 of the Regulation (EU) 2016/679 (General Data Protection Regulation). Code.org relies on derogations for specific situations as set forth in Article 49 of the GDPR. In particular, Code.org collects and transfers to the U.S. Personal Information only: with your consent; to perform a contract with you; or to fulfill a legitimate interest of Code.org in a manner that does not outweigh your rights and freedoms. Code.org endeavors to apply suitable safeguards to protect the privacy and security of your Personal Information and to use it only consistent with your relationship with Code.org and the practices described in this Privacy Policy. Please see “For EEA Users” below for more information about your Data Subject rights.
+
 
 ### Conditions of Use and Notices
 
 If you choose to visit our Website, your visit and any dispute over privacy is subject to this Privacy Policy and our Terms of Service, including limitations on damages, arbitration of disputes, application of the law of the State of Washington and venue for all disputes in Seattle, Washington.
 
 If any provision of this Privacy Policy or our Terms of Service is held invalid by a court of competent jurisdiction, the remainder of this Privacy Policy and our Terms of Service will remain in full force and effect. 
 
-## No Commercial Use of Student Data and How to Delete Student Data
+## No Commercial Use of Student Data
+
+Some laws, such as California’s Student Online Personal Information Protection Act (SOPIPA), prohibit the gathering of the Personal Information of K-12 students. Code.org abides by such laws and shall not use, disclose, or compile any student Personal Information for the purpose of marketing or advertising commercial products or services. We do not disclose any student Personal Information to third parties for marketing purposes.
+
+## You can review or delete your Personal Information at any time
+
+We keep your personal data  only as long as necessary or for the purposes for which it was collected or as required under any contract or by applicable law. 
+
+If, at any point, you wish to access, amend, erase, export (i.e., data portability), or object to or restrict the processing of Personal Information collected via our Website, send us a request at [https://code.org/contact](https://code.org/contact). We will promptly review all such requests in accordance with applicable laws. 
+
+We also automatically delete any Personal Information associated with inactive student or teacher accounts per our data retention policy. 
```

Comment:
I didn't know we did this! What is our time out limit? Do we send an email (if we have one for the Student or the Teacher's email) warning them that we will do this soon?